### PR TITLE
Auth: Move LDAP debug to Authentication menu

### PR DESCRIPTION
**Pull request by Jguer**
**What is this feature?**

- Move LDAP debug menu to an authentication card

image

When LDAP is disabled card won't show

image

This keeps the current behavior until extra LDAP configuration is added

**Why do we need this feature?**

[Add a description of the problem the feature is trying to solve.]

**Who is this feature for?**

[Add information on what kind of user the feature is for.]

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

Please check that:
- [x] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: pkg/api/api.go [hunk 1]
```diff
@@ -649,3 +646,10 @@ func (hs *HTTPServer) registerRoutes() {
 	r.Get("/api/snapshots-delete/:deleteKey", reqSnapshotPublicModeOrSignedIn, routing.Wrap(hs.DeleteDashboardSnapshotByDeleteKey))
 	r.Delete("/api/snapshots/:key", reqSignedIn, routing.Wrap(hs.DeleteDashboardSnapshot))
 }
+
+func evalAuthenticationSettings() ac.Evaluator {
+	return ac.EvalAny(ac.EvalAll(
+		ac.EvalPermission(ac.ActionSettingsWrite, ac.ScopeSettingsSAML),
+		ac.EvalPermission(ac.ActionSettingsRead, ac.ScopeSettingsSAML),
+	), ac.EvalPermission(ac.ActionLDAPStatusRead))
+}
```
Comment (eleijonmarck) [inline pkg/api/api.go:655]: one issue that this PR raises is to have the ScopeSettingsLDAP ready. could we add that as a follow up issue ?

**Review by eleijonmarck — APPROVED** [review]
LGTM, this could be merged before a big refactor of the UI from this PR. Ideally we would merge this PR first and then we could merged the conflicts into the bigger PR with the refactored states

oss - https://github.com/grafana/grafana/pull/71200
enterprise - https://github.com/grafana/grafana-enterprise/pull/5398

File: pkg/api/api.go (hunk 1 above)
Comment (Jguer) [reply pkg/api/api.go:655]: :+1: you want me to do it?

**Comment by mhollstein** [comment]
I don't see the Authentication under Administration menu, so I'm not be able to use it.
Grafana 10.1.0 Docker Image: docker pull grafana/grafana:10.1.0

**Comment by mari-arondeus** [comment]
> I don't see the Authentication under Administration menu, so I'm not be able to use LDAP Test user mapping. Grafana 10.1.0 Docker Image: docker pull grafana/grafana:10.1.0

Ditto. Maybe it only shows up if you have enterprise licensing so the SAML option would also be available? We're running Grafana Enterprise on-prem 10.1.0 without licensing, and the option is missing for us as well.

**Pull request merged**